tests/resources/conformance_suites/units/registry/utr-conf-cr-2013-05-17/2013-05-17/index.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Aribtrary line: We dont want to commit the unzipped version of these conformance suites. Please update this so its just the zip.